app/uk/gov/hmrc/incometaxproperty/connectors/error/ApiError.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incometaxproperty.connectors.error
+
+import play.api.libs.json.{JsValue, Json}
+
+case class ApiError(status: Int, body: ApiErrorBody) {
```

Comment:
we have this class already in the `models` package. Let's only have one. And if you keep this one, also bring the tests to the same package (we should have tests for this)

Reply:
Removed the one I created and retained the one that is on the main. Will check out the tests